fendermint/ipc/src/pof.rs

```diff
@@ -0,0 +1,85 @@
+// Copyright 2022-2023 Protocol Labs
+// SPDX-License-Identifier: Apache-2.0, MIT
+//! IPC proof of finality related functions
+
+use crate::Config;
+use crate::{BlockHeight, ParentViewProvider};
+use anyhow::anyhow;
+use ipc_sdk::cross::CrossMsg;
+use ipc_sdk::ValidatorSet;
+use serde::{Deserialize, Serialize};
+use std::sync::Arc;
+
+/// The proof for POF.
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Eq)]
+pub struct IPCParentFinality {
+    /// The latest chain height
+    pub height: u64,
+    /// The block hash. For FVM, it is a Cid. For Evm, it is bytes32.
+    pub block_hash: Vec<u8>,
+    /// new top-down messages finalized in this PoF
+    pub top_down_msgs: Vec<CrossMsg>,
+    /// latest configuration information from the parent.
+    pub config: ValidatorSet,
```

Comment:
```suggestion
    /// latest configuration information from the parent.
    pub validator_set: ValidatorSet,
```
You already call it `membership` in the interface, so that's a good name too, but `config` is such an overloaded term, and most of the time in this codebase it's about static settings of a component, not something as specific as the dynamic validator powers. I understand it's called _configuration_ in the theory papers, but in this context it's not helpful IMO.

Reply:
I renamed to `validator_set`, should be better.